src/generation.rs

```diff
@@ -95,7 +95,7 @@ impl Template {
         };
 
         // And now generate the files in the output dir given
-        let walker = WalkDir::new(&start_path)
+        let walker = WalkDir::new(&start_path).follow_links(definition.symlinks)
```

Comment:
Should we follow them by default?

Reply:
I wasn't sure of the original intent so I left it disabled by default, but I'm all for having it be the default case as long as it handles cycles during the walk and it doesn't effect Windows users. I'll test and makes sure that it doesn't introduce any issues there and if not make it the default.

Reply:
Initially the use case was for templates to be self contained, eg a git repo so there was no real need for symlinks